Description: fix tests
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: no
Bug-Debian: https://bugs.debian.org/762115
Last-Update: 2014-09-29

--- a/glance_store/tests/fakes.py
+++ b/glance_store/tests/fakes.py
@@ -13,9 +13,10 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+from glance_store import driver
 from glance_store import exceptions
 
 
-class UnconfigurableStore(base.Store):
+class UnconfigurableStore(driver.Store):
     def configure(self):
         raise exceptions.BadStoreConfiguration()
